packages/astro/package.json

```diff
@@ -44,6 +44,7 @@
     "components",
     "dist",
     "astro.js",
+    "app.d.ts",
```

Comment:
Nit, but any strong feelings on naming this `astro/env`? "app" means so many things, maybe `env` is more clear?

Reply:
Also, you probably need to add this to the `package.json#exports` field as well.

```json
{
  "exports": {
    "./app": "./app.d.ts"
  }
}
```

Reply:
No strong feelings regarding the name, I went with app just because I had to choose a name. `env` is better!

I'm able to refer to it through `tsconfig.json` and triple slashes comments without adding it to `exports`, so I figured it wasn't needed? I'll do research on this and see what other projects do

Reply:
Right, `typescript` doesn't respect `exports` field. In case they do in the future, it couldn't hurt to add explicitly export it. I've run into problems with tools _other than `typescript`_ trying to reference a `.d.ts` file that wasn't exported.